modules/shutup/src/main/PublicLine.scala

```diff
@@ -14,7 +14,7 @@ case class PublicLine(
 object PublicLine:
 
   def make(text: String, from: Source): PublicLine =
-    PublicLine(text, from.some, nowInstant.some)
+    PublicLine(text.take(200), from.some, nowInstant.some)
```

Comment:
Since forum posts can be quite long, only record the first 200 chars (chat message is 140), not to waste too much space.

May miss some posts where trigger words are put latter on but usually enough to have a good overview. Otherwise a smarter system could be put in place, only recording trigger words and their context.